Header header

# general contours
float64 max_contour_area
float64 rectangle_angle
float64 rectangle_width
float64 rectangle_height
float64 ellipse_angle
float64 ellipse_width
float64 ellipse_height

# Convex Hull Features
float64 max_hull_area

# spatial moments
float64 m00
float64 m10
float64 m01
float64 m20
float64 m11
float64 m02
float64 m30
float64 m21
float64 m12
float64 m03

# central moments
float64 mu20
float64 mu11
float64 mu02
float64 mu30
float64 mu21
float64 mu12
float64 mu03

# central normalized moments
float64 nu20
float64 nu11
float64 nu02
float64 nu30
float64 nu21
float64 nu12
float64 nu03

# center of mass m10/m00, m01/m00
float64 center_x
float64 center_y
float64 length
float64 area

